content/en/docs/contributing/development.md

```diff
@@ -26,6 +26,47 @@ cloud IDE:
 
 Gitpod automatically installs the repo-specific packages for you.
 
+## Cloning the Repository
+
+For first-time contributors, it is recommended to clone the repository using SSH instead of HTTPS to avoid common errors related to authentication.
+
+### Cloning with SSH
+
+To clone the repository using SSH, follow these steps:
+
+1. **Generate an SSH Key (if you don't have one):**
+   - Open a terminal and run:
+     ```bash
+     ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
```

Comment:
GitHub supports `ed25519` and `ecdsa` just fine, so there is no reason to use rsa.

```suggestion
     ssh-keygen -C "your_email@example.com"
```